server/src/db/adapter.js

```diff
@@ -0,0 +1,7 @@
+const RethinkDBAdapter = require('js-data-rethinkdb').RethinkDBAdapter;
+const dbinfo = require('../../config').database;
```

Comment:
camelCase?

Reply:
Good catch. I feel like Prettier should've caught that.